scriptoocloud/pom.xml

```diff
@@ -19,7 +19,7 @@
 		<admin.password>$2a$04$oQo44vqcDIFRoYKiAXoNheurzkwX9dcNmowvTX/hsWuBMwijqn44i</admin.password>
 
 		<db.driver>com.mysql.jdbc.Driver</db.driver>
-		<db.database.url>jdbc-url-to-db</db.database.url>
+		<db.database.url>db-url</db.database.url>
```

Comment:
let's no change this; it's unnecessary and just has the potential to create unnecessary conflicts.